### PR TITLE
Update convert-user-mailbox-to-shared-mailbox.md

### PR DESCRIPTION
**Pull request by msbemba**
Added another step to convert mailbox from admin center directly instead of exchange admin center.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 1]
```diff
@@ -67,6 +67,39 @@ Here's how you convert a mailbox to a shared mailbox:
...
+
+::: moniker range="o365-21vianet"
+
+1. Go to the [Office 365 admin center](https://login.partner.microsoftonline.cn).
+
+::: moniker-end
+
+2. In the admin center, expand **Users** \> **Active Users**.
+
+3. Choose the user whose mailbox nees to be converted
```
Comment (JohanFreelancer9) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
3. Choose the user whose mailbox needs to be converted.
```

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 2]
```diff
@@ -67,6 +67,39 @@ Here's how you convert a mailbox to a shared mailbox:
...
+
+4. Expand Mail settings, Under **More Settings**, click **Convert to Shared mailbox**.
```
Comment (JohanFreelancer9) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
4. Expand Mail settings. Under **More Settings**, click **Convert to Shared mailbox**.
```

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 3]
```diff
@@ -67,6 +67,39 @@ Here's how you convert a mailbox to a shared mailbox:
     
 6. For everything else you need to know about shared mailboxes, please see [Create a shared mailbox](create-a-shared-mailbox.md).
 
+## Convert the mailbox using the office365 admin center
```
Comment (JohanFreelancer9) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
## Convert the mailbox using the office365 admin center
```
```suggestion
## Convert the mailbox using the office 365 admin center
```

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 4]
```diff
@@ -67,6 +67,39 @@ Here's how you convert a mailbox to a shared mailbox:
...
+
+- Mailboxes can be converted directly from the admin center as well
```
Comment (JohanFreelancer9) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
- Mailboxes can be converted directly from the admin center as well.
```

**Comment by mypil** [comment]
@msbemba - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@cmcatee-MSFT - Can you please assist to check, approve and merge the content of this PR based on issue #528?

Thank you.

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 5]
```diff
@@ -67,6 +67,39 @@ Here's how you convert a mailbox to a shared mailbox:
     
 6. For everything else you need to know about shared mailboxes, please see [Create a shared mailbox](create-a-shared-mailbox.md).
 
+## Convert the mailbox using the office 365 admin center
+
+- Mailboxes can be converted directly from the admin center as well.
+
+::: moniker range="o365-worldwide"
+
+1. Go to the [Microsoft 365 admin center](https://admin.microsoft.com/AdminPortal/Home#/homepage).
```
Comment (cmcatee-MSFT) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=834822" target="_blank">Active users</a> page.
```

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 6]
```diff
@@ -67,6 +67,39 @@ Here's how you convert a mailbox to a shared mailbox:
...
+
+::: moniker-end
+
+::: moniker range="o365-germany"
+
+1. Go to the [Office 365 admin center](https://portal.office.de/adminportal/home).
```
Comment (cmcatee-MSFT) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=847686" target="_blank">Active users</a> page.
```

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 7]
```diff
@@ -67,6 +67,39 @@ Here's how you convert a mailbox to a shared mailbox:
...
+
+::: moniker-end
+
+::: moniker range="o365-21vianet"
+
+1. Go to the [Office 365 admin center](https://login.partner.microsoftonline.cn).
```
Comment (cmcatee-MSFT) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=850628" target="_blank">Active users</a> page.
```

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 8]
```diff
@@ -67,6 +67,39 @@ Here's how you convert a mailbox to a shared mailbox:
...
+
+::: moniker-end
+
+2. In the admin center, expand **Users** \> **Active Users**.
```
Comment (cmcatee-MSFT) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]: Delete this line

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 9]
```diff
@@ -67,6 +67,39 @@ Here's how you convert a mailbox to a shared mailbox:
...
+
+3. Choose the user whose mailbox needs to be converted.
```
Comment (cmcatee-MSFT) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
2. Choose the user whose mailbox you want to convert.
```

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 10]
```diff
@@ -67,6 +67,39 @@ Here's how you convert a mailbox to a shared mailbox:
...
+
+4. Expand Mail settings, Under **More Settings**, click **Convert to Shared mailbox**.
```
Comment (cmcatee-MSFT) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
3. In the right pane, choose **Mail**. Under **More actions**, choose **Convert to shared mailbox**.
```

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 11]
```diff
@@ -67,6 +67,39 @@ Here's how you convert a mailbox to a shared mailbox:
...
+
+5. If the mailbox is smaller than 50GB, you can remove the [license from the user](../subscriptions-and-billing/remove-licenses-from-users.md), and stop paying for it. Don't delete the user's old mailbox. The shared mailbox needs it there as an anchor. If you are converting the mailbox of an employee that is leaving your organization, you should take additional steps to make sure that they cannot log in anymore. Please see [Remove a former employee from Office 365](../add-users/remove-former-employee.md).
```
Comment (cmcatee-MSFT) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
4. If the mailbox is smaller than 50GB, you can remove the [license from the user](../subscriptions-and-billing/remove-licenses-from-users.md), and stop paying for it. Don't delete the user's old mailbox. The shared mailbox needs it there as an anchor. If you are converting the mailbox of an employee that is leaving your organization, you should take additional steps to make sure that they cannot log in anymore. See [Remove a former employee from Office 365](../add-users/remove-former-employee.md).
```

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 12]
```diff
@@ -67,6 +67,39 @@ Here's how you convert a mailbox to a shared mailbox:
...
+    
+6. For everything else you need to know about shared mailboxes, please see [Create a shared mailbox](create-a-shared-mailbox.md).
```
Comment (cmcatee-MSFT) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
5. For everything else you need to know about shared mailboxes, see [Create a shared mailbox](create-a-shared-mailbox.md).
```

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md (hunk 8 above)
Comment (msbemba) [reply Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]: This line is deleted now. We should be good to merge.

**Pull request merged**